synapse/storage/__init__.py

```diff
@@ -39,12 +40,15 @@ class Storage(object):
     """
 
     def __init__(self, hs, stores: DataStores):
+        self.stores = stores
+
         # We include the main data store here mainly so that we don't have to
         # rewrite all the existing code to split it into high vs low level
         # interfaces.
         self.main = stores.main
 
         self.persistence = EventsPersistenceStore(hs, stores)
+        self.state = StateGroupStorage(hs, stores)
```

Comment:
As per https://github.com/matrix-org/synapse/pull/6240#discussion_r340275090: the inconsistency in naming between Storage and Store is... inconsistent.